eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Core.hs

```diff
@@ -12,6 +16,12 @@ import Cardano.Ledger.BaseTypes (Network)
 import Cardano.Ledger.Keys (KeyHash (..), KeyRole (..))
 import Cardano.Ledger.SafeHash (SafeHash)
 import Cardano.Ledger.ShelleyMA.Core
+  ( Era (..),
+    EraIndependentScriptIntegrity,
+    EraTxBody (..),
+    ShelleyMAEraTxBody,
+  )
+import qualified Cardano.Ledger.ShelleyMA.Core as X
```

Comment:
Why is this change necessary? Is it just to be explicit about all the import used in this module?